docs/modules/install/pages/manual.adoc

```diff
@@ -121,6 +121,11 @@ bin/rails db:create db:migrate
 bin/rails db:seed
 ----
 
+[NOTE]
```

Comment:
I've just realized that we should change the initial phrase of "We should now create your database:" (line 116). How about "We should now create your database. For a first local installation, we recommend to start with some example contents (also known as seeds)."